src/rgw/rgw_trim_bilog.cc

```diff
@@ -527,7 +679,8 @@ int BucketTrimInstanceCR::operate(const DoutPrefixProvider *dpp)
           ldpp_dout(dpp, 0) << "WARNING: no connection to zone " << zid << ", can't trim bucket: " << bucket << dendl;
           return set_cr_error(-ECANCELED);
         }
-        using StatusCR = RGWReadRESTResourceCR<StatusShards>;
+
+	using StatusCR = RGWReadRESTResourceCR<StatusShards>;
```

Comment:
this call to `StatusCR` should be passing an extra param `{"version", "2"}`

Reply:
Oops, I had that in one version and didn't commit it. Thank you.